### PR TITLE
Header "component"

### PR DESCRIPTION
**Pull request by kosamari**
This is ugly and not according to the design. This is cor FYI

### CODE REVIEW

**Comment by jakearchibald** [comment]
@kosamari I've resolved the conflicts with this

**Comment by jakearchibald** [comment]
@googlebot would you ever fuck off

**Comment by jakearchibald** [comment]
@surma @kosamari this is ready for review

File: confbox.config.js [hunk 1]
```diff
@@ -7,9 +7,9 @@ module.exports = {
   /**
    * Start of conference in the above timezone, in the format: YYYY/MM/DD HH:mm.
    */
-  start: '2020/11/11 09:00',
+  start: '2019/11/11 09:00',
```
Comment (jakearchibald) [inline confbox.config.js]: I seriously put the wrong year in here first time around

Comment (surma) [reply confbox.config.js]: Ahahaha I wanted to leave a comment on review, but I thought that was intentional

File: src/_includes/header/style.css [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+  padding-bottom: 1px;
+
+  @media (min-width: 720px) {
+    padding: 0.5em 1em;
+    position: sticky;
+    top: 0;
+    display: grid;
+    grid-template-columns: [title] max-content 2.6em [reg] 1fr [extra-links] max-content;
+    align-items: center;
+    justify-items: start;
```
Comment (surma) [inline src/_includes/header/style.css]: Just as an FYI: You can combine this `{align,justify}-items` to `place-items: center start`.

File: src/_includes/header/style.css [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+}
+
+.media-fill {
+  composes: media-fill from '../utils.css';
+}
+
+.root {
+  background: #fff;
+  position: relative;
+  z-index: 1;
```
Comment (surma) [inline src/_includes/header/style.css]: Is z-index needed outside the MQ?

File: src/_includes/header/index.njk [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+        <img class="{% className css, 'media-fill' %}" src="confboxAsset(/assets/chromelogo.svg)" alt="Chrome">
+      </div>
+    </div>
+    <div class="{% className css, 'title-text' %}">
+      <div class="{% className css, 'title-main' %}">dev summit</div>
+      <span class="{% className css, 'title-year' %}">{% confDate conf.start, 'YYYY' %}</span>
+    </div>
+  </div>
+  <div class="{% className css, 'extra-links' %}">
+    <a href="#">Shedule</a>
```
Comment (surma) [inline src/_includes/header/index.njk]:
```suggestion
    <a href="#">Schedule</a>
```

File: src/_includes/header/style.css (hunk 2 above)
Comment (jakearchibald) [reply src/_includes/header/style.css]: Ta, I'll add that. I haven't internalised all the shorthands yet.

File: src/_includes/header/style.css (hunk 3 above)
Comment (jakearchibald) [reply src/_includes/header/style.css]: Yeah, for the register button that overlaps the section

File: src/_includes/header/index.njk (hunk 4 above)
Comment (jakearchibald) [reply src/_includes/header/index.njk]: lol I always fuck this one up

**Pull request merged**